sdks/java/maven-archetypes/examples/pom.xml

```diff
@@ -43,9 +43,37 @@
       </extension>
     </extensions>
 
+
+    <resources>
+      <resource>
```

Comment:
I added some comments. I am happy to explain it, but first, I woud like to test, if it understandable only from the comment ;-) Let me know if it's still not clear.